{
    "type": "bundle",
    "id": "bundle--ec2a962d-bde8-4f13-854d-5d60c50202b5",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--edf10e2e-e6a4-4329-b52f-815772635cbc",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-05-09T00:28:39.163174Z",
            "modified": "2024-05-09T00:28:39.163174Z",
            "name": "CVE-2024-4418",
            "description": "A race condition leading to a stack use-after-free flaw was found in libvirt. Due to a bad assumption in the virNetClientIOEventLoop() method, the `data` pointer to a stack-allocated virNetClientIOEventData structure ended up being used in the virNetClientIOEventFD callback while the data pointer's stack frame was concurrently being \"freed\" when returning from virNetClientIOEventLoop(). The 'virtproxyd' daemon can be used to trigger requests. If libvirt is configured with fine-grained access control, this issue, in theory, allows a user to escape their otherwise limited access. This flaw allows a local, unprivileged user to access virtproxyd without authenticating. Remote users would need to authenticate before they could access it.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-4418"
                }
            ]
        }
    ]
}